Models.js

```diff
@@ -0,0 +1,136 @@
+function getExp(skillName){
+  /* Gets exp of a skill from storage.
+   */
+  return new Promise((resolve, reject) => {
+    chrome.storage.sync.get([skillName], function (result) {
+      var overallExp;
+      if (skillName in result) {
+        overallExp = result[skillName];
+      }
+      else{
+        overallExp = 0;
+      }
+      resolve(overallExp);
+    });
+  });
+}
+
+function getLevelAndLevelUpExp(exp) {
+  /* Calculates current level and exp needed to next level.
+   * O(max_level) - computational complexity
+   */
+  var level = 0;
+  while(exp >= expTable[level + 1]){
+    level++;
+  }
+  var levelExp = exp - expTable[level];
+  var totalExpNeeded = expTable[level + 1] - expTable[level];
+  var levelUpExp = totalExpNeeded - levelExp;
+  var returnObject = [level, levelUpExp];
+  return returnObject;
+}
+
+function addSkill () {
+  /* Adds skill to storage and to current table.
+   */
+  var skillName = $('#skill_name').val();
+  $('#skill_name').val('');
+
+  var skillsDict = {};
+  skillsDict[skillName] = 0;
+  allSkills.push(skillName);
+
+  chrome.storage.sync.set({skillsArrayId: allSkills});
+  chrome.storage.sync.set(skillsDict);
+  debugAddingSkill(skillName);
+  skillToTable(allSkills.length - 1);
```

Comment:
I don't like this line. I guess the function from view shouldn't be in models.

Reply:
I can: 
1. Move this line to handleSkillButtons;
2. Create new function in Utils which will run addSkills and this line;
What other options do I have and which one do you want me to apply here?
Current solution seems best to the code but may be inconsistent with the MVC principles. The fault in this may lay that handleSkillButtons is currently in Models.

Reply:
I think this line should be moved to handleSkillButtons. If it wouldn't work, then leave it here. We'll think about where it should be placed later.